lib/schema.js

```diff
@@ -70,6 +70,11 @@ const load = (
         if (file === 'domains.schema') {
           for (const name in schema) {
             const domain = schema[name];
+            const decorator = domain.constructor.name;
+            if (decorator === 'Flags' && domain.enum) {
+              const enumDomain = schema[domain.enum];
```

Comment:
We will do second pass after schema loading https://github.com/metarhia/metaschema/issues/92 and it will link domains to categories and cross-link categories